readme.md

```diff
@@ -150,6 +161,10 @@ $breakpoints: (
   @media #{md('xs', 'max')} { // @media not all and (min-width: 48em) { ... }
     display: flex;
   }
+
+  @media #{md('s', 'min', 'em', 'height')} { // @media (min-height: 36em) { ... }
```

Comment:
We could add an example using `arglist` to pass arguments to a function:

```sass
@media #{md(( breakpoint: 's', orientation: 'height' )...)} {
  // ...
}
```

See [this article](https://www.sitepoint.com/sass-multiple-arguments-lists-or-arglist/) for more detailed information about `arglist`.

Reply:
That's a good idea. I'll adapt the others media function calls examples too.